### PR TITLE
Add Godot (GDScript) LSP instructions

### PR DESCRIPTION
**Pull request by lucypero**

### CODE REVIEW

File: docs/src/language_servers.md [hunk 1]
```diff
@@ -166,7 +166,24 @@ Follow installation instructions on [LSP-gopls](https://github.com/sublimelsp/LS
...
+    "clients": {
+        "godot-lsp": {
+            "enabled": true,
+            "command": ["/PATH/TO/godot-editor.exe"], // Update the PATH
+            "tcp_port": 6008,
+            "selector": "source.gdscript",
+        }
+    }
+}
+```
```
Comment (rwols) [inline docs/src/language_servers.md:186]: This code block should be indented by 4 spaces to make it a part of the bullet point.

**Review by rwols — CHANGES_REQUESTED** [review]
Thanks! I have a minor formatting nitpick.

**Comment by lucypero** [comment]
> Thanks! I have a minor formatting nitpick.

Done

**Review by rwols — APPROVED** [review]

**Comment by rwols** [comment]
Wouldn't this also require a step to install some Godot scripting language syntax?

**Comment by lucypero** [comment]
> Wouldn't this also require a step to install some Godot scripting language syntax?

Yes, you need to install the GDScript plugin to get syntax highlighting.

**Comment by rwols** [comment]
Would it be possible for you to add that as “step 1” in the instructions? Other instructions mention that as well, if the syntax is not shipped with ST.

**Comment by lucypero** [comment]
> Would it be possible for you to add that as “step 1” in the instructions? Other instructions mention that as well, if the syntax is not shipped with ST.

https://github.com/sublimelsp/LSP/pull/2005

**Pull request merged**